README.md

```diff
@@ -49,6 +49,7 @@ following:
 * `GADTs`
 * `InstanceSigs`
 * `KindSignatures`
+* `NoStarIsType`
```

Comment:
And `QuantifiedConstraints`? Or is that your separate `ShowSing` patch?

Reply:
Indeed, I put that in #352.